libraries/node-core-library/src/test/FileSystem.test.ts

```diff
@@ -43,10 +43,9 @@ describe(FileSystem.name, () => {
     });
 
     test('Should return true for an error on a file descriptor call', () => {
-      const buffer: Buffer = Buffer.allocUnsafeSlow(1024);
       expect.assertions(1);
       try {
-        fs.readSync(11, buffer, 0, buffer.length, -1);
+        fs.readFileSync(`${__dirname}/nonexistent.txt`);
```

Comment:
@iclanton , this test is now no longer testing the category of issue the description says it is.